west.yml

```diff
@@ -180,14 +180,14 @@ manifest:
     - name: nrf-802154
       repo-path: sdk-nrf-802154
       path: nrf-802154
-      revision: ee38cef4fd93ed2807f16580754057e3943ab1f5
+      revision: d1fe788bb15ff83c2b675cbd8b6e976c192e5c5b
```

Comment:
This doesnt look right. The sha is the same as the dragoon one.

Reply:
Yeah, must be a bug in the script